solid2/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width">
+        <title>Solid2 Hello World</title>
+        <link rel="stylesheet" href="https://cdn.simplecss.org/simple.min.css">
+    </head>
+
+    <body>
+        <h1>Solid2 Hello World</h1>
+
+        <div id="loading">
+            <p>Loading...</p>
+        </div>
+
+        <div id="auth-guest" hidden>
+            <p>Hi there!</p>
+            <p>
+                This page is a showcase of a simple <a href="https://solidproject.org/" target="_blank">Solid application</a>
+                built using JavaScript <a href="https://github.com/jeff-zucker/solid-file-client">solid-file-client library</a>. You can look at the source code and learn how to use it in
```

Comment:
I guess this should say "using the JavaScript library solid-file-client" or something else?